libs/contentful-extensions/mideind-translation/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "contentful-mideind-extension",
+  "version": "1.0.0",
+  "private": true,
+  "devDependencies": {
+    "@jeremybarbet/contentful-extension-scripts": "0.22.7",
+    "@contentful/eslint-config-extension": "0.4.2",
+    "@babel/core": "7.12.3",
+    "@babel/plugin-transform-runtime": "7.12.1",
+    "@babel/preset-env": "7.12.1",
+    "@babel/preset-react": "7.12.1",
+    "@testing-library/react": "11.1.1",
+    "cssnano": "4.1.10",
+    "contentful-cli": "1.4.48",
+    "eslint": "7.12.1",
+    "typescript": "4.0.5",
+    "@types/jest": "26.0.15",
+    "@types/react": "16.9.56",
+    "@types/react-dom": "16.9.9",
+    "@types/webpack-env": "1.15.3"
+  },
+  "scripts": {
+    "start": "contentful-extension-scripts start --output build/ --id mideind-translation --name mideind-translation --space-id 8k0h54kbe6bj --field-types Object",
```

Comment:
Are the two packages.json required? You could move these scripts inside the workspace.json so you could start/build and so on from the root

Reply:
Hmm.

The start and build scripts could be moved into the workspace. The remaining issue would be the deployment script.
The deploy script which we need in order to update the extension hosted on Contentful.
The deploy script relies on the `contentful-cli`.
If we were to move the deployment into the workspace then we would need to add this package or fork it and do some minimalization work.

I tested adding the `contentful-cli` package to our repo and that results in 19 new packages.

Deployments are going to be so infrequent that I'd consider keeping the package.json independent in this case, since these extensions, in particular, are not being deployed from within our system.

Is there perhaps something I am not seeing?

Reply:
Good point, better to keep it local to the extensions 👍